projects/discussion-ui/src/lib/components/tag-all-discussion/tag-all-discussion.component.html

```diff
@@ -9,10 +9,11 @@ <h2 class="margin-remove-bottom tagtitle" [ngStyle]="getBgColor(tagName)">{{tagN
         <div class="flex flex-1 flex-column margin-fix">
             <lib-app-loader *ngIf="fetchSingleCategoryLoader"></lib-app-loader>
 
-            <ng-container *ngIf="similarPosts && similarPosts.length > 0">
-                <ng-container *ngFor="let data of similarPosts">
+            <ng-container *ngIf="similarPosts  && similarPosts.length > 0">
+                <div (click)="navigateToDiscussionDetails(data);logTelemetry($event)"
```

Comment:
This was not there earlier?

Reply:
No navigation was there in tag related discussion. I added now.